.eslintrc

```diff
@@ -3,15 +3,21 @@
         "node": true,
         "jest": true
     },
-    "parser": "babel-eslint",
+    "parser": "@babel/eslint-parser",
```

Comment:
I updated dependencies while I was trying to work out exactly what was going wrong.